mmv1/third_party/terraform/services/dataflow/resource_dataflow_flex_template_job_test.go.erb

```diff
@@ -299,8 +299,9 @@ func TestAccDataflowFlexTemplateJob_withKmsKey(t *testing.T) {
 
 	randStr := acctest.RandString(t, 10)
 	job := "tf-test-dataflow-job-" + randStr
-	key_ring := "tf-test-dataflow-kms-ring-" + randStr
-	crypto_key := "tf-test-dataflow-kms-key-" + randStr
+	kms := acctest.BootstrapKMSKeyInLocation(t, "us-central1")
+	key_ring := kms.KeyRing.Name
```

Comment:
```suggestion
```

you'll also need to not pass this in to the config functions.

Reply:
Could you remove this variable as well? It should be unused at this point.

Reply:
I see `key_ring` being used in `Config: testAccDataflowFlexTemplateJob_kms(job, key_ring, crypto_key, bucket, topic),` below.

More importantly, my largest blocker with this PR and this repository, in general, is that it's not clear where `TestAccDataflowFlexTemplateJob_withKmsKey` even appears in the generated Go code. For example, in https://github.com/hashicorp/terraform-provider-google, searching for `TestAccDataflowFlexTemplateJob_withKmsKey` comes up empty. (See https://github.com/search?q=repo%3Ahashicorp%2Fterraform-provider-google%20TestAccDataflowFlexTemplateJob_withKmsKey&type=code).

Reply:
> I see key_ring being used in Config: testAccDataflowFlexTemplateJob_kms(job, key_ring, crypto_key, bucket, topic), below.

Yes, but this PR removes its usage _inside_ that function, so it is unused. You need to remove it here, in the function call, and in the function signature.

>  it's not clear where TestAccDataflowFlexTemplateJob_withKmsKey even appears in the generated Go code. For example, in https://github.com/hashicorp/terraform-provider-google, searching for TestAccDataflowFlexTemplateJob_withKmsKey comes up empty.

This test file is beta-only, so the entire file has a "version guard" that makes the contents only get rendered in beta: https://github.com/GoogleCloudPlatform/magic-modules/blob/c71562c01cbe257065593bfdf616d9dfacd8aefd/mmv1/third_party/terraform/services/dataflow/resource_dataflow_flex_template_job_test.go.erb#L3 For more information, see https://googlecloudplatform.github.io/magic-modules/develop/promote-to-ga/

Here's links the downstream file in the [`google`](https://github.com/hashicorp/terraform-provider-google/blob/main/google/services/dataflow/resource_dataflow_flex_template_job_test.go) and [`google-beta`](https://github.com/hashicorp/terraform-provider-google-beta/blob/38a557e635cb4dd45a55af9dad761fd444d1fd02/google-beta/services/dataflow/resource_dataflow_flex_template_job_test.go#L294) providers.